### DISCUSSION TITLE
Sort selected nodes by selection order, not by data tree order

### DISCUSSION THREAD

**Discussion by cywhale** [dowjones/react-dropdown-tree-select#435] [Q&A]
**Is your feature request related to a problem? Please describe.**
To make a simple sort/preferences of selections by users. For examples, 
data.json
 - A
 ---- A1
 ---- A2
- B
  ---- B1
  ---- B2
- C
if user select A1, C,  B2 in order,  I can pop out these selections by order. But by current design, the selection is always A1, B2, C

**Describe the solution you'd like**
Have an option to enable showing selected nodes by selection order. 

**Describe alternatives you've considered**
I can have an internal state to record order of selection by useState (in react), but I think to make this order of selection appear on the dropdown selected area much more make sense.

Draggable function is a more user-friendly design for this option, but maybe make things much bigger and complicated.

**Comment by mrchief** [comment]
Showing selections in order makes sense and it could be the default behavior. I don't see a need to have another preference for this. 

> Draggable function is a more user-friendly design for this option, but maybe make things much bigger and complicated.

Yeah, it complicates things for very little gains.

**Comment by luluwheelan** [comment]
I just submitted a similar question, I'd like the selected item ordered by selection order as well. 
Anyone can tell me how to achieve it? Thanks!
